src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Caching.cs

```diff
@@ -315,9 +315,16 @@ public bool Equals(JsonSerializerOptions? left, JsonSerializerOptions? right)
                     left._typeInfoResolver == right._typeInfoResolver &&
                     CompareLists(left._converters, right._converters);
 
-                static bool CompareLists<TValue>(ConfigurationList<TValue> left, ConfigurationList<TValue> right)
+                static bool CompareLists<TValue>(ConfigurationList<TValue>? left, ConfigurationList<TValue>? right)
                     where TValue : class?
                 {
+                    // NB equates null with empty lists
```

Comment:
What does `NB` stand for?

Reply:
nota bene

Reply:
Would it make the code more readable to just use `NOTE:`?

Reply:
Removed it since it's probably redundant when it comes to comments.